Remove `udp-services` ConfigMap from k8gb helm chart templates (#354)
AbsaOSS/k8gb: docs/deploy_infoblox.md

```

* Modify the example configuration. Important parameters described below:
  * `dnsZone` - this zone will be delegated to the `edgeDNS` in your environment. E.g. `yourzone.edgedns.com`
  * `edgeDNSZone` - this zone will be automatically configured by k8gb to delegate to `dnsZone` and will make k8gb controlled nodes act as authoritative server for this zone. E.g. `edgedns.com`
  * `edgeDNSServer` stable DNS server in your environment that is controlled by edgeDNS provider e.g. Infoblox so k8gb instances will be able to talk to each other through automatically created DNS names
  * `clusterGeoTag` to geographically tag your cluster. We are operating `eu` cluster in this example
  * `extGslbClustersGeoTags` contains Geo tag of the cluster(s) to talk with when k8gb is deployed to multiple clusters. Imagine your second cluster is `us` so we tag it accordingly

>   * `infoblox.enabled: true` to enable automated zone delegation configuration at edgeDNS provider. You don't need it for totally local test and can optionally skip it. Meanwhile in this how-to we will cover a fully operational end-to-end scenario.

Reviewer: ```suggestion
  * `infoblox.enabled: true` to enable automated zone delegation configuration at edgeDNS provider. You don't need it for local testing and can optionally be skipped. Meanwhile, in this section we will cover a fully operational end-to-end scenario.
```

make infoblox-secret
```

> * Configure NGINX Ingres Controller in order to expose DNS UDP port for `k8gb-coredns` service on working nodes.

Reviewer 2: These recommendations are not isolated to Infoblox only. They should be also provided for a [Quickstart](https://www.k8gb.io/#quick-start) and also for a [local setup](https://www.k8gb.io/docs/local.html) in-depth doc.
In case of local setup we might just incorporate config map patching/creating within the associated Make target

Author: @ytsarev, good point, will amend make targets 👍

Author: @ytsarev, amended. Also created DNS configuration document.

make infoblox-secret
```

* Expose associated k8gb CoreDNS service for DNS traffic on worker nodes.

>   > Check [this document](./exposing_dns.md) for detailed informationon.

Reviewer 2: ```suggestion
  > Check [this document](./exposing_dns.md) for detailed information.
```

Author: Thanks, amended
